network-api/networkapi/wagtailpages/tests/buyersguide/test_products.py

```diff
@@ -245,3 +314,86 @@ def test_cannot_be_created_more_than_two_levels_deep(self):
             'Categories can only be two levels deep.',
             form.errors['parent']
         )
+
+    def test_related_articles(self):
+        cat = BuyersGuideProductCategory.objects.create(name="Test category")
+        article1 = buyersguide_factories.BuyersGuideArticlePageFactory()
+        article2 = buyersguide_factories.BuyersGuideArticlePageFactory()
+        article3 = buyersguide_factories.BuyersGuideArticlePageFactory()
+        buyersguide_factories.BuyersGuideProductCategoryArticlePageRelationFactory(
+            category=cat,
+            article=article2,
+            sort_order=0,
+        )
+        buyersguide_factories.BuyersGuideProductCategoryArticlePageRelationFactory(
+            category=cat,
+            article=article1,
+            sort_order=1,
+        )
+        buyersguide_factories.BuyersGuideProductCategoryArticlePageRelationFactory(
+            category=cat,
+            article=article3,
+            sort_order=2,
+        )
+
+        related_articles = cat.related_article_relations.related_items()
+
+        self.assertEqual(len(related_articles), 3)
+        self.assertQuerysetEqual(
+            related_articles,
+            [article2, article1, article3],
+            ordered=True,
+        )
+
+    # TODO: Figure out why this test is failing
```

Comment:
This is the test that is producing a duplicate item and causing an error since 4 != 3. 

However, I added the skip flag for now per request of Tyler